src/Engine/TranskribusClient.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Engine;
+
+use App\Exception\OcrException;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TranskribusClient
+{
+
+    /** @var HttpClientInterface */
+    private $httpClient;
+
+    /** @var string transkribus access token. */
+    private $accessToken = '';
+
+    /** @var $config. */
+    private $config = [];
+
+    /** @var string Transcribed result. */
+    private const PROCESSES_URL = "https://transkribus.eu/processing/v1/processes";
+
+    /**
+     * TranskribusClient constructor.
+     * @param HttpClientInterface $httpClient
+     * @param string $accessToken
+     */
+    public function __construct(
+        HttpClientInterface $httpClient,
+        string $accessToken
+    ) {
+        $this->httpClient = $httpClient;
+        $this->accessToken = $accessToken;
+    }
+
+    /**
+     * @param string $imageURL
+     * @param mixed[] $config
+     * @return int
+     */
+    public function initProcess(string $imageURL, array $config = []): int
```

Comment:
Is `$config` ever used? It looks like we might want to pass an `$htrId` here instead, and use that in the config (which wouldn't need to be conditionally constructed).

Reply:
Noted, I think we need a parameter for content. This holds image co-ordinates.

Reply:
That's true, but it's not being used yet so there's no need to add it yet. It can be refactored as required.